build/parser.mjs

```diff
@@ -941,11 +941,20 @@ class Parser {
    * Adds releaseDate, vaultDate and estimatedVaultDate to all primes using
    * data from "Ducats or Plat".
    * @param {Item} item data to append vault data to
+   * @param {module:warframe-items.Category} category of the data
+   * @param {WikiaData} wikiaData from wikia to apply
    * @param {VaultData} vaultData to look up data for the {@param item}
    */
-  addVaultData(item, vaultData) {
+  addVaultData(item, category, wikiaData, vaultData) {
```

Comment:
i'm not sure why the category param is needed, we should already have the category defined from the item

also, generally you add new params at the end of the signature, not in the middle, unless you're adding required parameters before optional parameters.... but all of these are required

Reply:
It's not but because I don't understand javascript as much a fair bit was done from what I understood in `addAdditionalWikiaData`, I did want to originally use `item.category` or `item.type` but because I didn't see it being used I assumed there was a reason for it and just added it here, which is about the same explanation for the order actually

Reply:
ok, what about the stuff that can be vaulted? you can't vault resources...

Reply:
skill issues on my part then. but yeah I got you

Reply:
nono, this is definitely a cultural js thing that's developed over time that's not really well communicated

Reply:
still missing something because now there's no archwings in wikiadata